src/main/java/io/appium/java_client/remote/AppiumCommandExecutor.java

```diff
@@ -192,7 +192,11 @@ private Response createSession(Command command) throws IOException {
     }
 
     public void refreshAdditionalCommands() {
```

Comment:
if this method is not used anymore then it would probably make sense to mark it deprecated